meson.build

```
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+project('debugcc',
+  'c',
+  license: ['BSD-3-Clause'],
+  meson_version : '>= 0.61.0', # for install_symlink
+  default_options: [
+    'buildtype=release',
+    ])
```

My gut feeling says:

```suggestion
  default_options: [
    'buildtype=release',
  ])
```

But then maybe the closing `)` should also be on a separate line, and that's more like what `rustfmt` does than what might be the standard for meson? Fine either way.

---

done